.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk: oraclejdk8
+
+script:
+  - mvn test --batch-mode
+  - mvn checkstyle:check --batch-mode --fail-never
```

Comment:
Progress is progress. My :+1: stands. Failing the build on style failure will be more progress, and I look forward to 👍 ing that too.